lvs/__init__.py,sha256=aT6coGsVTx7Oai-9Ry4MjR6B5EUO4nErwk053AA66eQ,81
lvs/client.py,sha256=eCS50_OkJunQ0AT80qhgOX12xrf8udTJikefyEtv1OE,2604
lvs/exceptions.py,sha256=wYV0udD74RO9Cs0ULdqpSjkUxhepRZJOjgbOgvRBRec,148
lvs/settings.py,sha256=Fegx8xny9wLMAv6urP57fYyrC3n-0qaj-0TB4ICBHeg,482
cc_lvs-2.1.0.dist-info/METADATA,sha256=XaOqlearE50vFgXx921Rgf2fVSudg3W9Q-qC7zhoDOA,258
cc_lvs-2.1.0.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
cc_lvs-2.1.0.dist-info/top_level.txt,sha256=K_kQa17eb9jRWXKIqY2S_8TPrD9ubHp1gDq7OFvIhiY,4
cc_lvs-2.1.0.dist-info/RECORD,,
